src/Core/tests/UnitTests/WeakListTests.cs

```diff
@@ -18,7 +18,8 @@ public async Task ObjectsAreEvicted_GetEnumerator()
 			GC.Collect();
 			GC.WaitForPendingFinalizers();
 
-			foreach (var _ in list) ;
+			foreach (var _ in list)
+				;
```

Comment:
hmm can this be express on editorconfig ?

Reply:
If someone else wants to fix that, I'll probably just put a `{ }` block with a comment inside.